src/main/java/wolox/training/models/Book.java

```diff
@@ -113,6 +126,7 @@ public int getPages() {
     }
 
     public void setPages(int pages) {
+        Preconditions.checkNotNull(pages, "Pages cannot be null");
```

Comment:
Here we could add a check that controls if the number entered in the pages is greater than 0

Reply:
Add new validate greater than 0 in set